extensions/stats/plugin.h

```diff
@@ -135,6 +136,7 @@ struct IstioDimensions {
   // destination_version="v1",
   // destination_workload="svc01-0-8",
   // destination_workload_namespace="service-graph01",
+  // destination_port = "80",
```

Comment:
I realize the comment is not correct from the regex. Maybe you want to remove the SPACE character

Reply:
done